wpiutil/src/main/native/include/wpi/jni_util.h

```diff
@@ -399,12 +399,14 @@ class JSpanBase {
     return {reinterpret_cast<U*>(arr.data()), arr.size()};
   }
 
+  //! @cond Doxygen_Suppress
```

Comment:
What warning does Doxygen generate for these private member variables?

Reply:
/home/runner/work/allwpilib/allwpilib/wpiutil/src/main/native/include/wpi/jni_util.h:403: error: Found ';' while parsing initializer list! (doxygen could be confused by a macro call without semicolon)